bin/infra

```diff
@@ -73,6 +73,12 @@ pullEnv(){
     mv new.dev.env server/dev.env
 }
 
+
+runPrivelaged(){
```

Comment:
```suggestion
runPrivileged(){
```

Reply:
I suck at spelling - I don't know how my vscode plugin didn't catch this lol.